### PR TITLE
Language/Dashboard: migrate js alert and js confirm to modal

### PR DESCRIPTION
**Pull request by aeng**
https://zanata.atlassian.net/browse/ZNTA-1286
Based off: https://github.com/zanata/zanata-server/pull/1259

---
This change is [Reviewable](https://reviewable.io/reviews/zanata/zanata-server/1258)

<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/zanata/zanata-server/pull/1258%23issuecomment-236081054%22%2C%20%22https%3A//github.com/zanata/zanata-server/pull/1258%23issuecomment-236471152%22%2C%20%22https%3A//github.com/zanata/zanata-server/pull/1258%23issuecomment-236838837%22%2C%20%22https%3A//github.com/zanata/zanata-server/pull/1258%23discussion_r72915502%22%2C%20%22https%3A//github.com/zanata/zanata-server/pull/1258%23discussion_r72915889%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/zanata/zanata-server/pull/1258%23issuecomment-236081054%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Cimg%20class%3D%5C%22emoji%5C%22%20title%3D%5C%22%3Algtm%3A%5C%22%20alt%3D%5C%22%3Algtm%3A%5C%22%20align%3D%5C%22absmiddle%5C%22%20src%3D%5C%22https%3A//reviewable.io/lgtm.png%5C%22%20height%3D%5C%2220%5C%22%20width%3D%5C%2261%5C%22/%3E%20-%28Awaiting%20review%29%20%2BReviewed%20%2B%28On%20QA%29%20%5Cn%5Cn---%5Cn%5CnReviewed%204%20of%204%20files%20at%20r1.%5CnReview%20status%3A%20all%20files%20reviewed%20at%20latest%20revision%2C%20all%20discussions%20resolved.%5Cn%5Cn---%5Cn%5Cn%5Cn%5Cn%2AComments%20from%20%5BReviewable%5D%28https%3A//reviewable.io%3A443/reviews/zanata/zanata-server/1258%23-%3A-KNoXS-yFbvFtbfiJlzP%3A7167892%29%2A%5Cn%3C%21--%20Sent%20from%20Reviewable.io%20--%3E%5Cn%22%2C%20%22created_at%22%3A%20%222016-07-29T02%3A44%3A36Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1005842%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/huangp%22%7D%7D%2C%20%7B%22body%22%3A%20%22Dashboard%5Cr%5Cn-%20%5Bx%5D%20Leave%20language%20team%5Cr%5Cn-%20%5Bx%5D%20Reset%20API%20key%5Cr%5Cn-%20%5Bx%5D%20Remove%20connected%20account%5Cr%5Cn%5Cr%5CnLanguages%5Cr%5Cn-%20%5Bx%5D%20Remove%20language%5Cr%5Cn%22%2C%20%22created_at%22%3A%20%222016-08-01T01%3A34%3A20Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/4597195%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/djansen-redhat%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40djansen-redhat%20fixed%20up%20the%202%20comments%20from%20you.%22%2C%20%22created_at%22%3A%20%222016-08-02T08%3A30%3A26Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/766762%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/aeng%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%2055a694eec9d406520b597809700d963e7487b670%20zanata-war/src/main/resources/messages.properties%2030%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/zanata/zanata-server/pull/1258%23discussion_r72915889%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22This%20should%20be%20fixed%20as%20well%20-%20%5C%22this%20language%5C%22%20%28not%20language%20this%29.%20Better%20if%20the%20language%20name%20could%20be%20a%20part%20of%20the%20string.%22%2C%20%22created_at%22%3A%20%222016-08-01T01%3A28%3A04Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/4597195%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/djansen-redhat%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20zanata-war/src/main/resources/messages.properties%3AL1111-1118%22%7D%2C%20%22Pull%2055a694eec9d406520b597809700d963e7487b670%20zanata-war/src/main/webapp/WEB-INF/layout/dashboard/settings.xhtml%2060%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/zanata/zanata-server/pull/1258%23discussion_r72915502%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22This%20should%20be%20jsf.LeaveLanguageTeam%22%2C%20%22created_at%22%3A%20%222016-08-01T01%3A14%3A29Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/4597195%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/djansen-redhat%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20zanata-war/src/main/webapp/WEB-INF/layout/dashboard/settings.xhtml%3AL255-268%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/zanata/zanata-server/pull/1258#issuecomment-236081054'>General Comment</a></b>
- <a href='https://github.com/huangp'></a> :lgtm: -(Awaiting review) +Reviewed +(On QA)
---
Reviewed 4 of 4 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.
---
*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-server/1258#-:-KNoXS-yFbvFtbfiJlzP:7167892)*

- <a href='https://github.com/djansen-redhat'></a> Dashboard
- [x] Leave language team
- [x] Reset API key
- [x] Remove connected account
Languages
- [x] Remove language
- <a href='https://github.com/aeng'></a> @djansen-redhat fixed up the 2 comments from you.
- [x] <a href='#crh-comment-Pull 55a694eec9d406520b597809700d963e7487b670 zanata-war/src/main/webapp/WEB-INF/layout/dashboard/settings.xhtml 60'></a> &nbsp;<b><a href='https://github.com/zanata/zanata-server/pull/1258#discussion_r72915502'>File: zanata-war/src/main/webapp/WEB-INF/layout/dashboard/settings.xhtml:L255-268</a></b>
- <a href='https://github.com/djansen-redhat'></a> This should be jsf.LeaveLanguageTeam
- [x] <a href='#crh-comment-Pull 55a694eec9d406520b597809700d963e7487b670 zanata-war/src/main/resources/messages.properties 30'></a> &nbsp;<b><a href='https://github.com/zanata/zanata-server/pull/1258#discussion_r72915889'>File: zanata-war/src/main/resources/messages.properties:L1111-1118</a></b>
- <a href='https://github.com/djansen-redhat'></a> This should be fixed as well - "this language" (not language this). Better if the language name could be a part of the string.

<a href='https://www.codereviewhub.com/zanata/zanata-server/pull/1258?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/zanata/zanata-server/pull/1258?approve=1'></a>&nbsp;<a href='https://github.com/zanata/zanata-server/pull/1258'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by huangp** [comment]
:lgtm: -(Awaiting review) +Reviewed +(On QA) 

---

Reviewed 4 of 4 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-server/1258#-:-KNoXS-yFbvFtbfiJlzP:7167892)*

File: zanata-war/src/main/webapp/WEB-INF/layout/dashboard/settings.xhtml [hunk 1]
```diff
@@ -237,13 +255,13 @@
                 <span class="txt--understated l--push-left-quarter">
                   [#{locale.localeId.id}]
                 </span>
-                <a4j:commandLink
-                  styleClass="l--float-right txt--danger reveal__target"
+                <zanata:ajax-command-button styleClass="txt--danger button button--small reveal__target"
+                  title="#{msgs['jsf.dashboard.settings.removeAccount.label']}"
```
Comment (djansen-redhat) [inline zanata-war/src/main/webapp/WEB-INF/layout/dashboard/settings.xhtml]: This should be jsf.LeaveLanguageTeam

File: zanata-war/src/main/resources/messages.properties [hunk 2]
```diff
@@ -1111,6 +1111,7 @@ jsf.language.members.empty=No members
 jsf.language.member.joined=You are a member of this language
 jsf.language.requests=Requests
 jsf.language.myRequests=My Requests
+jsf.language.delete.title=Delete language
 jsf.language.delete.description=Are you sure you wish to delete language this from Zanata?
```
Comment (djansen-redhat) [inline zanata-war/src/main/resources/messages.properties]: This should be fixed as well - "this language" (not language this). Better if the language name could be a part of the string.

**Comment by djansen-redhat** [comment]
Dashboard
- [ ] Leave language team
- [ ] Reset API key
- [ ] Remove connected account
Languages
- [ ] Remove language

**Comment by aeng** [comment]
@djansen-redhat fixed up the 2 comments from you.

**Comment by djansen-redhat** [comment]
Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-server/1258)*

**Comment by djansen-redhat** [comment]
:white_check_mark: Tested

**Pull request merged**